### ISSUE TITLE
Need Docs: Dependency on 'multer' when integrating keystone with existing express app

### ISSUE THREAD

**Issue by v1p** [keystonejs/keystone#1081]
Man this is an awesome node framework, getting a schema less CMS running in no time.
I was trying to integrate the latest 0.3.1 into my app. And almost spent many hours figuring out why POST data's body is going as {}.
Then I saw this issue #309 is asking to add multipart/form-data to all the forms. 
Now, if the parent app is not making use of multer or multipart/form-data parsing at all, this becomes an untraceable problem, unless we go put debug statements in views/list.js.

Perhaps if this can be added to documentation ? 

And oh yeah, I got keystone to work with my app (after adding multer ofcourse) & it's awesome!

**Comment by morenoh149** [comment]
did you see these? https://github.com/keystonejs/keystone/wiki/Knowledge-Base https://github.com/keystonejs/keystone/wiki/How-to-Add-Keystone-to-an-Already-Existing-Express-App
If you didn't we could refer to it more prominently in the docs. If you did could you add in solutions to the difficulties you had.

**Comment by v1p** [comment]
@morenoh149 : It'll be great if this is added to documentation more prominently, otherwise rest of the setup was fairly trivial, to get keystone working with an existing express app. 

I did referred to wiki & saw the example of setting the app up. 'multer', it wasn't clear that why it's required. Not every app has use of mutlipart/form(even if bodyParser is going to be deprecated w.r.t express 4, it doesn't justify to move every form to multipart/form-data type encoding), so I had to manually debug the middleware data flow to understand what was happening. 

I know I could have just copy pasted the whole example from the wiki, but from my past experiences in the copy/paste style of getting things done, I was not comfortable doing that. Only after reading through #309, I got to know that all admin forms have the encType set to multipart/form-data, and my middleware had no configuration to parse it.

Thus, after adding multer to express config (only required by keystone though, I have no multipart forms in my app), I got keystone to work flawlessly, with my existing app.
